packages/host/app/lib/matrix-model/room.ts

```diff
@@ -0,0 +1,44 @@
+import { tracked } from '@glimmer/tracking';
+
+import type { MatrixEvent as DiscreteMatrixEvent } from 'https://cardstack.com/base/matrix-event';
+import type {
+  RoomCreateEvent,
+  RoomNameEvent,
+} from 'https://cardstack.com/base/matrix-event';
+
+export class RoomModel {
+  @tracked events: DiscreteMatrixEvent[] = [];
+
+  get roomId() {
+    return this.events.length > 0 ? this.events[0].room_id : undefined;
+  }
+
+  get created() {
+    let event = this.events.find((e) => e.type === 'm.room.create') as
+      | RoomCreateEvent
+      | undefined;
+    if (event) {
+      return new Date(event.origin_server_ts);
+    }
+    return new Date();
```

Comment:
probably wanna carry over the comment from panel.gts about the newly created rooms not necessarily having a date so that it is clear to other devs why we are making a new date here